common/common.go

```diff
@@ -31,6 +31,7 @@ const (
 	MetadataServiceIP      = "169.254.169.255"
 	MetadataServicePubPort = 80
 	MetadataServicePrvPort = 2375
+	MetadataServiceRegSock = "/var/run/rkt-metadata-svc.sock"
```

Comment:
Why? As we're trying to keep rkt portable, I think it's best to stick with FHS. FHS defines /var/run for that and it doesn't have /run: http://refspecs.linuxfoundation.org/FHS_2.3/fhs-2.3.html
In practice, distros that have /run usually have them sym-linked so it doesn't matter.